src/keyboard.js

```diff
@@ -129,8 +135,10 @@ export function initKeys() {
  *
  * @param {String|String[]} keys - Key or keys to bind.
  * @param {(evt: KeyboardEvent) => void} callback - The function to be called when the key is pressed.
+ * @param {'keydown'|'keyup'} handler - Whether to bind to keydown or keyup events. Default: 'keydown'.
```

Comment:
The tests are failing in the tyepscript as this JSDoc is written as a required property. To make it optional use the `[]` syntax. If you also use the default sytanx of `=` you don't need to list the default in the comment:

```suggestion
 * @param {'keydown'|'keyup'} [handler=keydown] - Whether to bind to keydown or keyup events.
```